decentraland/components/presentational/LinkToEtherscan.test.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { shallow } from "enzyme";
+import LinkToEtherscan from "./LinkToEtherscan";
+
+describe("LinkToEtherscan", () => {
+  describe("renders the component", () => {
+    it("null account", async () => {
+      const account = null;
+      expect(shallow(<LinkToEtherscan account={account} />)).toMatchSnapshot();
```

Comment:
I'll be curious to see what the snapshot looks like in the null case below (just a note to self)

Reply:
Resolving this